Pod/Classes/GDGEntity.h

```diff
@@ -13,21 +12,21 @@
 
 @protocol GDGEntityFillDelegate <NSObject>
 
-- (void)entity:(GDGEntity*)entity hasFilledPropertyNamed:(NSString*)propertyName;
-- (void)entity:(GDGEntity*)entity requestToFillPropertyNamed:(NSString*)propertyName;
+- (void)entity:(GDGEntity *)entity hasFilledPropertyNamed:(NSString *)propertyName;
+
+- (void)entity:(GDGEntity *)entity requestToFillPropertyNamed:(NSString *)propertyName;
 
 @end
 
 @interface GDGEntity : NSObject
 
 @property (assign, nonatomic) NSUInteger id;
-@property (readonly, nonatomic) GDGEntityManager<GDGEntityFillDelegate>* db;
+@property (readonly, nonatomic) GDGEntityManager <GDGEntityFillDelegate> *db;
 
-- (BOOL)save;
-- (BOOL)drop;
++ (instancetype)entity;
```

Comment:
[https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSArray_Class/#//apple_ref/occ/clm/NSArray/array]([NSArray array])
[https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSDictionary_Class/#//apple_ref/occ/clm/NSDictionary/dictionary]([NSDictionary dictionary])
[https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSSet_Class/#//apple_ref/occ/clm/NSSet/set]([NSSet set])
[https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSString_Class/#//apple_ref/occ/clm/NSString/string]([NSString string])

Just following the standards